test/luatest_helpers/vtest.lua

```diff
@@ -7,7 +7,7 @@ local uuid = require('uuid')
 local yaml = require('yaml')
 local vrepset = require('vshard.replicaset')
 
-local wait_timeout = 120
+local wait_timeout = 50
```

Comment:
I do not insist on that change. However, on luatest versions of tests there's no output in case of hang due to timeout loop.

Reply:
I couldn't understand this part: `Moreover, it allows the timeout loop to be executed at least 2 times. If the test check the returned value more often than after 3 timeout loop, it will definitely notice the wrong value.` but ok anyway, agreed.